### ISSUE TITLE
Anaconda Prompt not installed

### ISSUE THREAD

**Issue by alexanderzimmerman** [ContinuumIO/anaconda-issues#8794]
This is related to issue #8784. Unfortunately, I cannot use the solution from #8420 because there is no Anaconda Prompt.

### Actual Behavior

I see no Anaconda Prompt in the Windows Start Menu under Anaconda3, and fail at searching for it.

### Expected Behavior

The installer should install the Anaconda Prompt, which should be available in the Windows Start Menu.

### Steps to Reproduce

- Install Windows 64 bit Anaconda 3 using the latest downloader and installer from the official page
- Select **Windows Start Menu > Anaconda 3**

##### Anaconda or Miniconda version:
https://repo.continuum.io/archive/Anaconda3-5.1.0-Windows-x86_64.exe

##### Operating System:
Windows 10 Pro
Version	10.0.16299 Build 16299

##### `conda info`
I can't do this, because the Anaconda Prompt is missing.

```

```

##### `conda list --show-channel-urls`
I can't do this, because the Anaconda Prompt is also missing.

```

```

**Comment by hhegedus** [comment]
I have this exact same problem.  Can someone please help?

**Comment by ck37** [comment]
I also have this problem :(

**Comment by deadletterb** [comment]
I think this is the solution: https://groups.google.com/a/continuum.io/forum/#!topic/anaconda/NuDydW1d7wg

When it gives you command line prompts to type in, be sure to run CMD in administrator mode.

**Comment by MichaelClerx** [comment]
Same issue here

**Comment by fsfkellner** [comment]
Same issue using Windows 10

**Comment by ToeKneeShoe** [comment]
I have this issue too and can't resolve it

**Comment by MichaelClerx** [comment]
Reinstalling helped

**Comment by yogeshsaswade** [comment]
Executing these two commands worked
    conda update menuinst
    conda install -f console_shortcut ipython ipython-notebook ipython-qtconsole launcher spyder
Source: https://groups.google.com/a/continuum.io/forum/#!topic/anaconda/NuDydW1d7wg

**Comment by dstanner** [comment]
I have the same problem.  Neither Anaconda Navigator nor the prompt installed. Even after adding the Anaconda folders to my path and rebooting, conda commands do not work.

@yogeshsaswade I can't use that fix because no conda commands work.

**Comment by pliantenergy** [comment]
> I have the same problem. Neither Anaconda Navigator nor the prompt installed. Even after adding the Anaconda folders to my path and rebooting, conda commands do not work.
> 
> @yogeshsaswade I can't use that fix because no conda commands work.

Same issue. No Anaconda Prompt, no Anaconda Navigator, no conda, nothing.   I've removed and reinstalled several times, made sure my system and user environment variables are clear of Anaconda and python related paths. There are no other python installations on my machine besides Abaqus python which, as far as I know is isolated and never has caused a problem before.

I'm trying to use the Windows 64 bit installer for Anaconda 2 2018.12.

No errors are thrown during installation. On the latest try, I downloaded and ran DebugView and the output consists only of messages from `SysTrayIcon` and `BackupServiceX` like this: 
```
00000017	310.88848877	[16476] 2019-03-07T18:29:34.74-05:00 # 16480:SysTrayIcon::UpdateIcon new state backingup	
00000018	315.93817139	[16476] 2019-03-07T18:29:39.79-05:00 # 16480:SysTrayIcon::UpdateIcon new state idle
```

and this: 

```
00000036	863.07708740	[16476] 2019-03-07T18:38:46.91-05:00 E 16316:ERROR: BackupServiceX::MonitorState connect closed 955 times since startup. Last error=0	
00000037	872.07769775	[16476] 2019-03-07T18:38:55.60-05:00 E 16076:ERROR: BackupServiceX::MonitorAlerts connect closed 961 times since startup. Last error=0
```
These messages also happen outside of the Anaconda Install.

**Comment by jannikbuhr** [comment]
Same problem here. Using win 10. I uninstalled and installed 64 bit anaconda3 several times and even tried the 32 bit version. The anaconda3 folder lacks the folder `bin` that previous versions had and none of the conda executables can be found

**Comment by AB-HardWorkBeatsNaturalTalent** [comment]
It would be useful if, when the anaconda is installing, and if there is no anaconda prompt or visibility in the start menu afterward,  the user sould see an audio/video clip : "my anaconda dont want none if you aint got buns hun" .. this would be more fun than just seeing the failed install by itself.

**Comment by dleeftink** [comment]
Same issue here when trying to install `Anaconda3-2018.12-Windows-x86_64`. No conda, no navigator. This was preceded by the the fatal mkl_intel_thread.dll error as described in #10213, after which I tried to reinstall Anaconda but without success. This all followed after having used the `conda init powershell` command and setting the environmental variables as described [here ](https://conda.io/projects/conda/en/latest/user-guide/troubleshooting.html#numpy-mkl-library-load-failed). 

Both CMD and Powershell started to return the error message `The system cannot find the path specified`, which I solved by following the solution outlined [here](https://www.tenforums.com/general-support/122582-cmd-terminal-starts-system-cannot-find-path-specified.html). Similar to #8460, Anaconda and Miniconda now fail to properly install. I would like to know what can be done to revert back to a working version.

**Comment by Excludos** [comment]
Still a problem. Just got a new computer, and Anaconda Navigator or prompt doesn't install, despite no errors. No previously installed versions. Tried 4 times and still nothing :(

**Comment by jamesoliverh** [comment]
This is also an issue for me. After installing Anaconda 3 64-bit in Windows 10 I see Anaconda Navigator but no Anaconda Prompt.

**Comment by RDxR10** [comment]
I was installing some new modules through Anaconda Prompt. It took time but somehow I think it got erased from my computer. After that, I could not find Anaconda Prompt.
The last thing I typed on it was this:
`conda env create -f environment.yml`

**Comment by ctivanovich** [comment]
This happened me today, after one month of using with no issues, suddenly today anaconda navigator is devoid of all apps and the anaconda prompt disappeared from search and the windows start menu. Windows 10. I can navigate to scripts and activate directly, but it's weird.

**Comment by JimShady** [comment]
This happened to me today also. Not sure what to do. Jupyter Lab is in my Start Menu, but no Anaconda Prompt.

**Comment by Arian96669** [comment]
Happened to me also a few minutes ago.  I had all my normal Jupyter and Spider shortcuts under the Anaconda3 Start menu folder, but no "Anaconda Prompt" shortcut.

I fixed it by opening my Anaconda Navigator, and then installing console_shortcut to my base(root) environment.

I now again have my "Anaconda Prompt" under my Windows 10 Start menu.  So everything fixed

**Comment by RyanBe123** [comment]
Just happened to me too, any solutions?

**Comment by vivabrian** [comment]
It just happened to me today too after doing a conda update.

**Comment by thegnord** [comment]
Arian96669 solution fixed for me. Hero!

**Comment by msarahan** [comment]
There are two issues here. One is with the shortcut not being installed in the first place. The cause there is most often windows start menu not refreshing, or accidental confusion with another python installation on the system at install time.

Next, the problem that people are setting more now is disappearing after an update. Please see https://github.com/conda/conda/issues/8842

**Comment by samarth0174** [comment]
Arian96669's  ,  Thanks ur soln worked for me 👍

**Comment by Arian96669** [comment]
> Arian96669's , Thanks ur soln worked for me 👍

Glad I could help!!

**Comment by lxt3** [comment]
Yep, same here after trying to install keras library for R.  But, I did a 'conda install console_shortcut' from a cmd prompt at the /condabin directory.  I really didn't want to have to reinstall anaconda3 due to issues I remember with getting tensorflow right.  Thanks!

**Comment by hongsoog** [comment]
Arian96669 solution  worked for me like charm. Thanks.

```sh
(base) C:\Users\hongsoog>conda install console_shortcut
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: C:\Users\kimkk_000\Anaconda3

  added / updated specs:
    - console_shortcut


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    console_shortcut-0.1.1     |                3         107 KB
    ------------------------------------------------------------
                                           Total:         107 KB

The following NEW packages will be INSTALLED:

  console_shortcut   pkgs/main/win-64::console_shortcut-0.1.1-3


Proceed ([y]/n)? y
```

**Comment by memememeda** [comment]
> This happened me today, after one month of using with no issues, suddenly today anaconda navigator is devoid of all apps and the anaconda prompt disappeared from search and the windows start menu. Windows 10. I can navigate to scripts and activate directly, but it's weird. I've gone ahead and added the anaconda3 scripts path to my windows environment path, which allows me to run activate from the regular DOS command line, but still, no idea why anaconda prompt disappeared, or why Navigator can no longer locate all the resources like Jupyter Lab and Spyder.

same situation, did u type  conda clean  recently?

**Comment by owen-synergy** [comment]
This happened to me after updating to conda 4.7.10 (with Miniconda - I was previously on 4.7.5) on Windows 10. Obviously don't have Anaconda Navigator so I had to find the conda.exe directory (C:\Users\<user name>\AppData\Local\Continuum\miniconda3\Scripts) - open a regular prompt to that directory and typed:

activate.bat base

to get to the base environment prompt then used Arian96669's solution to:

conda install console_shortcut

Now it all seems good.

**Comment by mukerong** [comment]
Same issue happened to me yesterday. I did `conda update conda-builds`. After finishing, both Anaconda Command Prompt and Anaconda Navigator disappear. Seems the updates remove them all!

Use @owen-synergy's command method to re-installed anaconda prompt, and then used anaconda prompt to run `conda install anaconda-navigator` to re-install navigator.

Can we make sure not to remove them in the future updates? I am not sure if anything else is removed. It is annoying to uninstall & reinstall again.

**Comment by AB-HardWorkBeatsNaturalTalent** [comment]
I resolved this issue locally by reverting to the previous version of the installer. I had struggled with this for a week or 2, being completely unable to install it no matter which sequence/configuration/uninstalling/reinstalling etc. I tried. But, using a previous installer was the magic sauce. I hope that is good enough for others too.

**Comment by mukerong** [comment]
@AB-HardWorkBeatsNaturalTalent, how did you find the previous version installer?

Seems some other packages got removed as well. When I load Jupyter, I got this error message. No issues were found before.

![image](https://user-images.githubusercontent.com/23180334/62403451-0e116200-b542-11e9-90c2-9955d7ae392c.png)

**Comment by jthemanoverthere** [comment]
Adrian96669 has a solution for this. Here's my tweak to his answer.

Steps:
1. Open Anaconda Navigator
2. Installing console_shortcut to base(root) environment / (or) udpate index
3. Your cmd should be open. I ran a test: conda install numpy and was ready to go
4. Pin Anaconda Prompt to taskbar before closing (to be on the safe side)

**Comment by SatvikNema** [comment]
Yeah man i was suffering from the same problem. And i figured out my anti virus was messing things up. If you get a popup form your anti-virus, DON'T close the window. Click on allow or whatever it asks you to do. Just don't ignore it. Or try disabling the antivirus and download Anaconda again. trust me it will work.

**Comment by IDNinja** [comment]
You should all try @Arian96669 's solution before further investigation.

**Comment by saiko85** [comment]
Hey Guys great news- Iv dicovered its the Antivirus that blocked both my Navigator and Prompt from being Installed- I saw it mentioned here- and it turns out to be why. Put the software on the exclusion list and your good to go ^_^

**Comment by yogeshsaswade** [comment]
I faced this issue again. After successfully installing anaconda 64 bit on windows 10, Anaconda prompt and navigator were not present in the start menu. I was getting the issue because of the Kaspersky antivirus.
Have disabled it for some time and reinstalled the anaconda, and the issue was gone.

**Comment by AnneWeir** [comment]
1. Open Anaconda Navigator and click on "Environment" tab. 
2. Search for "console_shortcut". 
a. If it's not installed, select it and install it (like Arian96669 mentions)
b. If it's already installed then uninstall it, and then re-install it again. (this worked for me)

**Comment by saiko85** [comment]
thanks- your right it was my antivirus- totally works now

**Comment by yus002** [comment]
> Executing these two commands worked
> conda update menuinst
> conda install -f console_shortcut ipython ipython-notebook ipython-qtconsole launcher spyder
> Source: https://groups.google.com/a/continuum.io/forum/#!topic/anaconda/NuDydW1d7wg

he said no prompt here. How do you run the code with no anaconda prompt

**Comment by inkyouin** [comment]
> I faced this issue again. After successfully installing anaconda 64 bit on windows 10, Anaconda prompt and navigator were not present in the start menu. I was getting the issue because of the Kaspersky antivirus.
> Have disabled it for some time and reinstalled the anaconda, and the issue was gone.

For those does not have the navigator and prompt, try this!!!!!
Disabled your antivirus fix the issue for me!!!!!!!!!!!! Thanks！

**Comment by gokbeykeskin** [comment]
In my case, the path I've installed Anaconda had non-English characters so the build failed. Removing anaconda and installing it in a different location solved the problem.
